### ISSUE TITLE
VaDatePicker update

### ISSUE THREAD

**Issue by asvae** [epicmaxco/vuestic-ui#1905]
We have Vasily design on new iteration of date picker: [figma](https://www.figma.com/file/2yH0KFiWds3q1B5NA5dELZ/vuestic-ui?node-id=1557%3A9995).

- [ ] Sort requirements in [notion](https://www.notion.so/epicmax/Vuestic-ui-10e698cacd5846688647e1d8482277c9?p=d6c42050b2d84faf8602cec051846d90).
- [ ] Create new issue with imlementation requirements.
- [ ] Assign someone to review this issue before starting with implementation.

**Comment by aluarius** [comment]
Moved to https://github.com/epicmaxco/vuestic-ui/issues/2148
